ui/src/timeMachine/components/view_options/ScatterOptions.tsx

```diff
@@ -0,0 +1,184 @@
+  getFillColumnsSelection,
+  getSymbolColumnsSelection,
+} from 'src/timeMachine/selectors'
+
+// Types
+import {ComponentStatus} from '@influxdata/clockface'
+import {Color} from 'src/types/colors'
+import {AppState} from 'src/types'
+
+interface StateProps {
+  fillColumns: string[]
+  symbolColumns: string[]
+  availableGroupColumns: string[]
+}
+
+interface DispatchProps {
+  onSetFillColumns: typeof setFillColumns
+  onSetSymbolColumns: typeof setSymbolColumns
+  onSetColors: typeof setColors
+  onSetYAxisLabel: typeof setYAxisLabel
+  onSetXAxisLabel: typeof setXAxisLabel
+  onUpdateAxisSuffix: typeof setAxisSuffix
+  onUpdateAxisPrefix: typeof setAxisPrefix
+}
+
+interface OwnProps {
+  xColumn: string
+  yColumn: string
+  fillColumns: string[]
+  symbolColumns: string[]
+  xDomain: [number, number]
+  yDomain: [number, number]
+  xAxisLabel: string
+  yAxisLabel: string
+  xPrefix: string
+  xSuffix: string
+  yPrefix: string
+  ySuffix: string
+  colors: Color[]
+  showNoteWhenEmpty: boolean
+}
+
+type Props = OwnProps & DispatchProps & StateProps
+
+const ScatterOptions: SFC<Props> = props => {
+  const {
+    fillColumns,
+    symbolColumns,
+    colors,
+    availableGroupColumns,
+    yAxisLabel,
+    xAxisLabel,
+    onSetFillColumns,
+    onSetSymbolColumns,
+    onSetColors,
+    onSetYAxisLabel,
+    onSetXAxisLabel,
+    yPrefix,
+    ySuffix,
+    onUpdateAxisSuffix,
+    onUpdateAxisPrefix,
+  } = props
+
+  const groupDropdownStatus = availableGroupColumns.length
+    ? ComponentStatus.Default
+    : ComponentStatus.Disabled
+
+  return (
+    <Grid.Column>
+      <h4 className="view-options--header">Customize Scatter Plot</h4>
+      <h5 className="view-options--header">Data</h5>
+
+      <Form.Element label="Symbol domain column">
+        <MultiSelectDropdown
+          selectedIDs={symbolColumns}
+          onChange={onSetSymbolColumns}
+          status={groupDropdownStatus}
+        >
+          {availableGroupColumns.map(columnName => (
+            <Dropdown.Item
+              id={columnName}
+              key={columnName}
+              value={{id: columnName}}
+            >
+              {columnName}
+            </Dropdown.Item>
+          ))}
+        </MultiSelectDropdown>
+      </Form.Element>
+      <Form.Element label="Fill domain column">
+        <MultiSelectDropdown
+          selectedIDs={fillColumns}
+          onChange={onSetFillColumns}
+          status={groupDropdownStatus}
+        >
+          {availableGroupColumns.map(columnName => (
+            <Dropdown.Item
+              id={columnName}
+              key={columnName}
+              value={{id: columnName}}
+            >
+              {columnName}
+            </Dropdown.Item>
+          ))}
+        </MultiSelectDropdown>
+      </Form.Element>
+      <h5 className="view-options--header">Options</h5>
+      <Form.Element label="Colors">
+        <ColorSchemeDropdown
+          value={colors.filter(c => c.type === 'scale')}
```

Comment:
This is an example of where I think using `string[]` instead of `ViewColor[]` in the `ScatterViewProperties` is an improvement. It seems confusing that colors have a `type` here. I think in reality it can only ever be one type?